firebase-app-distribution/firebase-app-distribution.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
     implementation project(path: ':firebase-components')
     implementation project(path: ':firebase-installations-interop')
     implementation project(path: ':firebase-common')
-    implementation project(path: ':firebase-installations')
+    runtimeOnly project(':firebase-installations')
```

Comment:
Our gradle has (path: something). I didn't see this in other gradle files so I am not sure why we have that. Decided to copy the runtimeOnly code that other projects have although it doesn't fit with our outline. Should we change our gradle file to remove those (path: ) code snippets not sure if they are necessary

Reply:
Yeah let's remove that since it seems unnecessary.